uber/models/__init__.py

```diff
@@ -674,7 +666,7 @@ def jobs_for_signups(self):
                 job.to_dict(fields)
                 for job in jobs if (job.required_roles or frozenset(job.hours) not in restricted_hours)]
 
-        def process_refund(self, txn):
+        def process_refund(self, stripe_log, model="Attendee"):
```

Comment:
Whoops, the default value for `model` is wrong here. I forgot to change it when I changed the implementation, which now uses the model itself rather than a string.